datadog_checks_base/datadog_checks/utils/decorator.py

```diff
@@ -0,0 +1,31 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+from datadog_checks.config import is_affirmative
+from ddtrace import tracer
+from functools import wraps
+from inspect import getargspec
+
+try:
+    import datadog_agent
+except ImportError:
+    # Integration Tracing is only available with Agent 6
+    pass
+
+def trace_func(func):
```

Comment:
Let's rename to `trace`, `tracer`, or `add_tracing`

Reply:
I'm not sure we want it to name it the same thing as an exposed function from ddtrace? I chose `trace_func` at the time since its just a decorator that performs a standard manual instrumentation step for any function.

Reply:
Perhaps `tracing` or `add_tracing` then?

Reply:
I went with `add_tracing`